asapdiscovery-data/asapdiscovery/data/tests/test_rdkit.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from asapdiscovery.data.backend.rdkit import load_sdf
+
+
+def test_ligand_sdf(moonshot_sdf, multipose_ligand, sdf_file):
+    single_conf = load_sdf(moonshot_sdf)
+    assert single_conf.GetNumConformers() == 1
+    multiconf = load_sdf(multipose_ligand)
+
+    assert multiconf.GetNumConformers() == 50
+
+    # this should fail if the file has multiple ligands
+    with pytest.raises(RuntimeError):
```

Comment:
Our openeye version doesn't fail, intead it just grabs the first one silently. Perhaps we should try and make behaviour consistent.

Reply:
I've made this an issue here #909